.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Description of the change
+<!--- Describe your changes in detail. -->
+
+
+## Reference
+<!--- Link to the reference implementation, research paper, and GitHub issue. -->
+
+## Demo Notebook
+<!-- If the change is related to a new model, provide an end-to-end working demonstration of the model usage, training, and numerics verification. -->
+
+## Checklist
+<!--- Please make sure all checkboxes are ticked before submitting this PR for review. -->
+
+- [ ] I have performed a self-review of my code.
```

Comment:
These first two checks (self review and comments) feel a little heavy handed, I'd just delete them.